src/renderer/features/wallets/ImportKeys/lib/import-keys-utils.ts

```diff
@@ -176,3 +196,33 @@ function mergeChainDerivations(
     duplicated: duplicatedKeys,
   };
 }
+
+const DERIVATION_ERROR_LABEL = {
+  [DerivationValidationError.INVALID_PATH]: 'dynamicDerivations.importKeys.error.invalidPath',
+  [DerivationValidationError.PASSWORD_PATH]: 'dynamicDerivations.importKeys.error.invalidPasswordPath',
+  [DerivationValidationError.MISSING_NAME]: 'dynamicDerivations.importKeys.error.missingName',
+  [DerivationValidationError.WRONG_SHARDS_NUMBER]: 'dynamicDerivations.importKeys.error.wrongShardsNumber',
+};
+
+function getErrorsText(t: TFunction, error: ValidationError, details?: ErrorDetails) {
+  switch (error) {
+    case ValidationError.INVALID_FILE_STRUCTURE:
+      return t('dynamicDerivations.importKeys.error.invalidFile');
+    case ValidationError.INVALID_ROOT:
+      return t('dynamicDerivations.importKeys.error.invalidRoot');
+    case ValidationError.DERIVATIONS_ERROR:
+      if (!details) return '';
+
+      return Object.keys(details).reduce<string>((acc, error) => {
+        const invalidValues = details[error as DerivationValidationError];
+        if (!invalidValues.length) return acc;
+
+        const errorText = t(DERIVATION_ERROR_LABEL[error as DerivationValidationError], {
+          count: invalidValues.length,
+          invalidValues: invalidValues.join(', '),
+        });
+
+        return `${acc}${errorText} `;
```

Comment:
Shouldn't be a divider between `acc` and `errorText`?

Reply:
The space after `errorText` is a divider. If we put it between `acc` and `errorText` we gonna have empty space before first error